server/common/utils/helpers.js

```diff
@@ -401,6 +401,7 @@ const fireAndForget = promise => promise.catch(err => console.error(err));
 
 const returnPayload = promise => payload => promise.then(() => payload);
 
+// TODO: Write test to this function
```

Comment:
Write test or remove the comment

Reply:
I know, that's why this comment is left here 😂